common/config/beachball/changelog-config.ts

```diff
@@ -2,7 +2,6 @@ import { BeachballConfig } from 'beachball';
 import { renderHeader, renderEntry } from './changelog-custom-renders';
 
 export const config: BeachballConfig = {
-  branch: 'origin/main',
```

Comment:
So this will cause beachball to use `main` but not just from `origin`?

Reply:
Indeed! If there is an upstream that matches the .git of the package .json it will use that instead automagically

screencap it auto-used the `upstream/main` without telling it to:
![image](https://user-images.githubusercontent.com/2684369/193103861-43abaed2-24d1-46db-8de1-a1b63e5f3c33.png)

Reply:
If your curious here's how:
![image](https://user-images.githubusercontent.com/2684369/193104563-555fb32c-5139-46f1-b556-090ca6909d01.png)